src/frontend/src/config.json

```diff
@@ -6,27 +6,5 @@
       "url": "https://d2z7d5345ccuw9.cloudfront.net/5c9bdffa37ed813dbe27f8f1/Workers_with_earnings_1250_per_month_or_less.grid",
       "showOnMap": true
     }
-  ],
```

Comment:
I meant to leave a comment in this JSON file, but: 1. I did not find a good way to comment a JSON file; 2. we might be removing the JSON file in dev phase.